openrtb-json/src/main/scala/com/powerspace/openrtb/json/BidResponseSerde.scala

```diff
@@ -1,20 +1,27 @@
 package com.powerspace.openrtb.json
 
 import com.google.openrtb.{BidResponse, NoBidReason}
-import io.circe.Decoder
+import io.circe.{Decoder, Encoder, Json}
 
+/**
+  * Serialize and Deserialize an OpenRTB BidResponse
+  */
 object BidResponseSerde {
+
   def decoder(implicit seatBidDecoder: Decoder[BidResponse.SeatBid]): Decoder[BidResponse] = {
     cursor =>
       for {
         id <- cursor.downField("id").as[String]
+        seatBids <- cursor.downField("seatbid").as[Seq[BidResponse.SeatBid]]
```

Comment:
seatBids can be empty (in case of nobid)

Reply:
oki-doki